File: app/src/main/java/info/nightscout/androidaps/plugins/PumpInsightLocal/app_layer/history/StopReadingHistoryMessage.java
package info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.history;

import info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.AppLayerMessage;
import info.nightscout.androidaps.plugins.PumpInsightLocal.app_layer.Service;
import info.nightscout.androidaps.plugins.PumpInsightLocal.descriptors.MessagePriority;
import info.nightscout.androidaps.plugins.PumpInsightLocal.utils.ByteBuf;

public class StopReadingHistoryMessage extends AppLayerMessage {

    public StopReadingHistoryMessage() {
        super(MessagePriority.NORMAL, false, false, Service.HISTORY);
    }
}

File: pump/insight/src/main/kotlin/app/aaps/pump/insight/app_layer/history/StopReadingHistoryMessage.kt
package app.aaps.pump.insight.app_layer.history

import app.aaps.pump.insight.app_layer.AppLayerMessage
import app.aaps.pump.insight.app_layer.Service
import app.aaps.pump.insight.descriptors.MessagePriority

class StopReadingHistoryMessage : AppLayerMessage(MessagePriority.NORMAL, false, false, Service.HISTORY)